src/Synapse/Synapse/help/Get-AzSynapseRoleAssignment.md

```diff
@@ -100,6 +100,20 @@ PS C:\> Get-AzSynapseRoleAssignment -WorkspaceName ContosoWorkspace -RoleDefinit
 This command gets a role assignment under workspace ContosoWorkspace with role name ContosoRole and user principal name ContosoName.
 
 ### Example 4
+```powershells
+PS C:\> Get-AzSynapseRoleAssignment -WorkspaceName ContosoWorkspace -RoleDefinitionName ContosoRole -SignInName ContosoName -ItemType ContosoItemType
```

Comment:
>ContosoItemType [](start = 134, length = 15)

I prefer to use a valid item type, for example `ApacheSparkPool`. But the current `ContosoItemType` is also fine. You can decide whether to change or not.